modules/core/src/loaders/xviz-loader-interface.js

```diff
@@ -174,6 +174,8 @@ export default class XVIZLoaderInterface {
 
   _getDataVersion = () => this.get('dataVersion');
 
+  _getStreamMetadata = () => this.get('streamMetadata');
```

Comment:
Also a similar suggestion to rename `_getStreamMetadata` to `_getStreamsMetadata ` and update to `this.get('streamsMetadata');`